### ISSUE TITLE
Configure scrolling (disable/offset)

### ISSUE THREAD

**Issue by sartwittig** [okta/okta-signin-widget#1754]
### I'm submitting a
  - [ ] bug report
  - [x] feature request

### Background info

We are using the Okta Sign-In Widget in one of our projects. The project also displays a Navbar with ~80px height fixed at the top of the viewport (Bootstrap navbar class + `position: fixed`, which shouldn't be too rare for a web application UI).

When the user enters incorrect credentials, the widget scrolls the viewport to have error message positioned at the top of the viewport. The message however is hidden by our navbar.

### Expected behavior

As the widget can not know if other elements of the applications are placed at the position it scrolls to, we would either need to configure the widget to not scroll at all (previously requested here: https://github.com/okta/okta-signin-widget/issues/365) or configure an offset, so it would be able to scroll the error message to appear just below the navbar.

### What went wrong?

Widget scrolls but error message is not visible due to our navbar being fixed to the top.

### Steps to reproduce

This fiddle illustrates the problem: https://jsfiddle.net/h681su9r/

The error message will be about the server connection but the navbar hides most of the message after scrolling. 

### Your environment

  - Okta Sign-In Widget Version: 5.4.3 (most recent version from npmjs.com)
  - Browser: Chrome 88.0.4324.190, Firefox 86.0.1
  - OS: Windows
  - Language: German/English

**Comment by denysoblohin-okta** [comment]
Thanks for submitting this issue.
Internal ref: https://oktainc.atlassian.net/browse/OKTA-379458

**Comment by aarongranick-okta** [comment]
@sartwittig Providing an option to disable the `scrollOnError` feature seems feasible. Would this change meet your needs? 

In other words, do you even need/want the widget to scroll the viewport? `scrollOnError` was added to accommodate very long (tall) forms. Usually the widget is only a couple hundred pixels high.

**Comment by sartwittig** [comment]
@aarongranick-okta yes, scrollOnError would help in our case. My previous fiddle was using your version 2.9.0 but I just forked it to check the behavior in the most recent version 5.7.3. It appears, that entering incorrect data does not trigger scrolling to the error message anymore. Most of the times it seems to scroll to the username input field:  https://jsfiddle.net/wpnegyq5/

**Comment by aarongranick-okta** [comment]
@sartwittig Does the behavior in version `5.7.3` meet your needs (does it work with the fixed banner on the page) ?

**Comment by sartwittig** [comment]
@aarongranick-okta I just upgraded to 5.7.3 in our project and here the scrolling is still happening when I enter invalid credentials. I've removed all customization from the config prior to trying it out and in the end the only difference to the scribble was, that we use @okta/okta-react as authClient (this shouldn't cause any scrolling AFAIK).

What would the expected behavior with your current implementation?
